src/OpenLoco/Platform/Platform.h

```diff
@@ -12,6 +12,7 @@ namespace OpenLoco::platform
     std::string promptDirectory(const std::string& title);
     fs::path GetCurrentExecutablePath();
     std::vector<fs::path> getDrives();
+    bool isRunningWine();
```

Comment:
```suggestion
    bool IsRunningInWine();
```

Reply:
We don't use InitialCamelCase for function names in OpenLoco ;)

Reply:
updated